kge/job/train.py

```diff
@@ -224,6 +226,9 @@ def _run(self) -> None:
 
             # create checkpoint and delete old one, if necessary
             self.save(self.config.checkpoint_file(self.epoch))
+            if self.epoch == 1 and not self.config.get("train.checkpoint.keep_init"):
+                # delete init checkpoint after first epoch
+                self._delete_checkpoint(0)
```

Comment:
I suggest to delete the init checkpoint as any other checkpoint (after keep new checkpoints) *except* if keep_init is true (then always keep it).